content/administration/maintain/supported_versions.rst

```diff
@@ -61,11 +62,8 @@ This matrix shows the support status of every version.
 
     N/A Never released for this platform
 
-    |orange| Some of our older customers may still run this version on our Odoo Online servers, we provide help only on blocking issues and advise you to upgrade.
-
```

Comment:
Is the removal of orange to "push" admins to upgrade their old versions?

If I'm correct, databases on really old versions _must_ upgrade soon. Wouldn't we make a distinction on this page and use red for these versions, and orange for everything else? 

(I assume we won't, but I still wanted to share my thoughts :) )